No need to run-depend on setuptools_rust
Relax delendencies to allow poetry-core 2.0 and setuptools_rust 1.11

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -223,7 +223,7 @@ pydantic = ">=1.7.4, <3"
 # This isn't really a dev-dependency, as `poetry install --no-dev` will fail,
 # but the alternative is to add it to the main list of deps where it isn't
 # needed.
-setuptools_rust = ">=1.3"
+# setuptools_rust = ">=1.3"
 
 # This is used for parsing multipart responses
 python-multipart = ">=0.0.9"
@@ -370,7 +370,7 @@ tomli = ">=1.2.3"
 # runtime errors caused by build system changes.
 # We are happy to raise these upper bounds upon request,
 # provided we check that it's safe to do so (i.e. that CI passes).
-requires = ["poetry-core>=1.1.0,<=1.9.1", "setuptools_rust>=1.3,<=1.10.2"]
+requires = ["poetry-core>=1.1.0", "setuptools_rust>=1.3"]
 build-backend = "poetry.core.masonry.api"
 
 
